importing/signals/handlers.py

```diff
@@ -13,6 +14,7 @@
 def set_object_permissions(sender, instance: PermissionsBase, **kwargs):
     """Set object-level permissions"."""
     instance.set_object_permissions()
+    ingest_data(instance.pk)
```

Comment:
For clarity, I'd probably either put this in it's own appropriately named signal handler or rename this one.